sqld/src/lib.rs

```diff
@@ -85,6 +86,7 @@ pub struct Config {
     pub rpc_server_ca_cert: Option<PathBuf>,
     pub enable_bottomless_replication: bool,
     pub create_local_http_tunnel: bool,
+    pub idle_shutdown_timeout: Option<Duration>,
```

Comment:
I've seen in the past `_s` or `_seconds` being added to time params name to make is more apparent what unit is used. Do you think it would make sense to add it to our name as well?

Reply:
Nvm. I see you did that on the command-line param.